app/helpers/staff_helper.rb

```diff
@@ -0,0 +1,21 @@
+module StaffHelper
+
+  def allow_rating?(proposal)
+    program_mode? || !proposal.has_speaker?(current_user)
+  end
+
+  def show_ratings?(rating)
+    program_mode? || rating.persisted?
+  end
+
+  def show_proposal?(proposal)
+    program_mode? || !proposal.has_speaker?(current_user)
+  end
+
+  def program_tracker
+    if program_mode?
+      hidden_field_tag(:program, true)
+    end
+  end
```

Comment:
I think there's a ton of consolidation in how we use Pundit, helpers and controller methods.  It'll be great to follow up with the other refactoring pieces.